berlin-school-of-economics-and-law-international-marketing-management.csl

```diff
@@ -0,0 +1,184 @@
+    <updated>2018-11-08T10:55:26+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="available at">online at URL</term>
+    </terms>
+  </locale>
+  <macro name="creator-year-long">
+    <group delimiter=" ">
+      <names variable="author">
+        <name delimiter="/" initialize-with=". " name-as-sort-order="all"/>
+        <label form="short" prefix=" (" suffix=")"/>
+        <substitute>
+          <names variable="editor"/>
+          <text variable="title"/>
+        </substitute>
+      </names>
+      <text macro="year" prefix="(" suffix=")"/>
+    </group>
+  </macro>
+  <macro name="editor-container-title">
+    <group delimiter=": ">
+      <text term="in"/>
+      <group delimiter=", ">
+        <names variable="editor">
+          <name delimiter="/" initialize-with=". " name-as-sort-order="all"/>
+          <label form="short" text-case="capitalize-first" prefix=" (" suffix=")"/>
+        </names>
+        <text variable="container-title"/>
+      </group>
+    </group>
+  </macro>
+  <macro name="publication-place-publisher">
+    <text variable="publisher-place"/>
+  </macro>
+  <macro name="creator-year-short">
+    <group delimiter=", ">
+      <text macro="creator-short"/>
+      <text macro="year"/>
+    </group>
+  </macro>
+  <macro name="edition">
+    <choose>
+      <if match="any" is-numeric="edition">
+        <group delimiter=" ">
+          <number variable="edition" form="ordinal"/>
+          <text term="edition" text-case="capitalize-first"/>
+        </group>
+      </if>
+      <else>
+        <text variable="edition"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="collection-title-number">
+    <group delimiter=" " prefix="(= " suffix=")">
+      <text variable="collection-title"/>
+      <text variable="collection-number"/>
+    </group>
+  </macro>
+  <macro name="page-range">
+    <group delimiter=" ">
+      <text term="page" form="short"/>
+      <text variable="page"/>
+    </group>
+  </macro>
+  <macro name="online-url">
+    <group delimiter=": ">
+      <text term="available at"/>
+      <group delimiter=" ">
+        <text variable="URL"/>
+        <date form="text" variable="accessed" prefix="(" suffix=")"/>
+      </group>
+    </group>
+  </macro>
+  <macro name="container-title-volume">
+    <group delimiter=", ">
+      <text variable="container-title"/>
+      <group delimiter=" ">
+        <number variable="volume" form="ordinal"/>
+        <text term="volume"/>
+      </group>
+      <group delimiter="/">
+        <number variable="issue"/>
+        <text macro="year"/>
+      </group>
+    </group>
+  </macro>
+  <macro name="creator-short">
+    <names variable="author">
+      <name form="short" delimiter="/"/>
```

Comment:
I'm guessing you'll want delimiter-precedes-last="always" here? I don't think this would otherwise draw a / between two authors.

Reply:
thanks for spotting those.